### ISSUE TITLE
Android?

### ISSUE THREAD

**Issue by Allineer** [stacks-task-manager/stacks#4]
Hello and thanks for this awesome app!

Android version is on ToDo column? :)

**Comment by synapse** [comment]
Yes, in order to create a phone app we would like to release the server first.
